temporalio/client.py

```diff
@@ -1950,8 +1956,10 @@ async def start_update(
         Args:
             update: Update function or name on the workflow.
             arg: Single argument to the update.
+            wait_for_stage: Required stage to wait until returning. ADMITTED is
```

Comment:
The first sentence of this comment is self-referential and adds no value beyond the name of the field.
Generally, I'd like there to be more clues in the python SDK about what these wait stages mean, in particular flagging that the worker must be present.  I'd suggest adding comments to the enum and/or linking out to docs.

Reply:
The headline 
> Send an update request to the workflow and return a handle to it
also just echoes what we already know from the name of the method and the return type.

I think there's an opportunity to headline its semantics (like a glance at what you'd find in the docs) and link out to overview docs.  

Context: A natural thing I've done when not finding any good description in the code is to find docs online.  But I might find python-specific docs which also don't provide a good overview of semantics.  As a n00b I might not even be aware that there are overview docs, as I don't have the ontology of temporal or its docs site in my head.  
Giving me the basics here and then linking out to the right docs page would save a lot of time and help me learn.

Reply:
> The first sentence of this comment is self-referential and adds no value beyond the name of the field.
This is often the case with required docs. When you are forced to document self-documenting things, the sentences become a bit redundant.

> Generally, I'd like there to be more clues in the python SDK about what these wait stages mean, in particular flagging that the worker must be present. I'd suggest adding comments to the enum and/or linking out to docs.

We try to avoid linking out to docs since those links are rarely stable (we do sometimes though). Want to take a stab at providing suggested documentation here?

Reply:
I don't think nearly as many things are self-documenting as engineers think are self-documenting.  Pretend like you're newish to temporal workflows and reading the comments about update.  What questions would you have?  What gotchas would you like to be made aware of?

I think you go too fast for me to keep up with you and write your comments, so maybe better for you to suggest and I'll take a look.

Reply:
My suggestion is what's already in this PR, heh, because it's consistent with the rest of the SDK. This has been sitting for many days already so I'm guessing there's no rush. The request is a bit too subjective, so If you can provide a suggestion then we'll know it's correct instead of guess-and-check.

Reply:
Per off-PR discussion, added link to https://docs.temporal.io/workflows#update